Library/Formula/cern-ndiff.rb

```diff
@@ -0,0 +1,16 @@
+
+class CernNdiff < Formula
+  desc "Numerical diff tool"
+  # Note: ndiff is a sub-project of Mad-X at the moment..
+  homepage "http://cern.ch/mad"
+  url "http://svn.cern.ch/guest/madx/tags/5.02.08/madX/tools/numdiff"
+  version "5.02.08"
+  head "http://svn.cern.ch/guest/madx/trunk/madX/tools/numdiff"
+
+  depends_on "cmake" => :build
+
+  def install
+    system "cmake", ".", *std_cmake_args
+    system "make", "install"
+  end
+end
```

Comment:
Can a test be added to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!